### DISCUSSION TITLE
Tabs component in documentation

### DISCUSSION THREAD

**Discussion by MimmyJau** [shadcn-ui/ui#2618] [Q&A]
Does shadcn come with a tab variant that looks like what they themselves use in their documentation? See video for what I'm talking about. It's quite different than the default tab component. 

https://github.com/shadcn-ui/ui/assets/45905800/1dfb0593-28f4-4738-9c9c-c9a881009f25

**Comment by ezeparziale** [comment]
Those tabs are easily achieved with Link, border-b and border-2 when the pathname is active.
